src/styles.scss

```diff
@@ -121,23 +125,22 @@
 }
 
 .react-kanban-column-header__button {
-  color: #333333;
-  background-color: #ffffff;
-  border-color: #cccccc;
-
-  &:hover,
-  &:focus,
-  &:active {
-    background-color: #e6e6e6;
-  }
+  @extend .react-kanban-form__button;
+  background-color: #fff;
 }
 
-.react-kanban-column-adder-button {
-  border: 2px dashed #eee;
-  height: 132px;
-  margin: 5px;
-
-  &:hover {
-    cursor: pointer;
+.react-kanban-column-header-form {
+  border-radius: 3px;
+  input {
```

Comment:
This is the same of `.react-kanban-column-header-form__title`, isn't? What if we don't use `.react-kanban-column-header-form__title`? This is going to be exported, we need to document this API on the docs.